code-challenges.test.js

```diff
@@ -51,12 +79,34 @@ const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deducti
 
 
 // b) Create the function that makes the test pass.
-
-
+// Pseudo Code:
+// Create a function called capitalWords with array as the param
+//  Create a variable inside the function called  upperCase
+// Use the .map method to take the strings out of the array
+// return the value and use .toUpperCase(), to upper case the first letters
+// Use .substring(1) to get the first letter of each string
+// return upperCase so the function will have an output
```

Comment:
Pseudocode is looking much better!  I want to clarify a few things though.  Map is going to iterate through each element in the array not necessarily take the strings out of the array.  On line 92 you are using .toUpperCase on the zero index of the current value as it is iterated on.  You are then adding substring(1) which is adding in the rest of the value back to the first letter upper cased.  The 1 that is being passed is the index on where you want to create the substring from the value.  So if value is 'streetlamp', value.substring(1) will be 'treetlamp' and value[0] is 's'.